system/Helpers/array_helper.php

```diff
@@ -15,6 +15,21 @@
 
 // CodeIgniter Array Helpers
 
+if (! function_exists('array_duplicate_by')) {
+    /**
+     * Duplicate array check by column.
+     *
+     * @param array|string $column Unique Column
+     * @param array        $data   Array Data
+     *
+     * @return array
+     */
+    function array_duplicate_by($column, array $data = [])
+    {
+        return ArrayHelper::arrayDuplicatesBy($column, $data);
+    }
```

Comment:
I would use names like the following for consistency with other array helpers.

```suggestion
    /**
     * Returns duplicate elements from an array by key(s).
     *
     * @param array|string $key   Key(s) to check
     * @param array        $array Array Data
     *
     * @return array
     */
    function array_duplicate_by($key, array $array)
    {
        return ArrayHelper::duplicatesBy($key, $array);
    }
```

Reply:
Done